pynhm/utils/parameters.py

```diff
@@ -218,6 +244,34 @@ def nhm_coordinates(self) -> dict:
             "nhm_seg": self.nhm_segment_coordinate,
         }
 
+    def parameters_to_json(self, json_filename):
+        """write the parameters dictionary out to a json file"""
+        json.dump(
+            self.parameters,
+            open(json_filename, "w"),
+            indent=4,
+            cls=JSONParameterEncoder,
+        )
+
```

Comment:
I kinda like `return` at the end of functions, if extra pedantic `return None`. I like them visually